pkg/jsonline/value_test.go

```diff
@@ -104,43 +104,43 @@ func TestValueMarshalString(t *testing.T) {
 		value    interface{}
 		expected interface{}
 	}{
-		{nil, `null`},
+		{nil, `<nil>`},
 		// signed integers
-		{int(-2), `"-2"`},
-		{int8(-1), `"-1"`},
-		{int16(0), `"0"`},
-		{int32(1), `"1"`}, // int32 is an alias of rune
-		{int64(2), `"2"`},
+		{int(-2), `-2`},
+		{int8(-1), `-1`},
+		{int16(0), `0`},
+		{int32(1), `1`}, // int32 is an alias of rune
+		{int64(2), `2`},
 		// unsigned integers
-		{uint(0), `"0"`},
-		{uint8(1), `"1"`},
-		{uint16(2), `"2"`},
-		{uint32(3), `"3"`},
-		{uint64(4), `"4"`},
+		{uint(0), `0`},
+		{uint8(1), `1`},
+		{uint16(2), `2`},
+		{uint32(3), `3`},
+		{uint64(4), `4`},
 		// floats
-		{float32(1.2), `"1.2"`},
-		{float64(-1.2), `"-1.2"`},
+		{float32(1.2), `1.2`},
+		{float64(-1.2), `-1.2`},
 		// complex numbers => NOT SUPPORTED YET
-		// {complex64(1.2i + 5), `"(5+1.2i)"`},
-		// {complex128(-1.0i + 8), `"(8-1i)"`},
+		// {complex64(1.2i + 5), `(5+1.2i)`},
+		// {complex128(-1.0i + 8), `(8-1i)`},
 		// booleans
-		{true, `"true"`},
-		{false, `"false"`},
+		{true, `true`},
+		{false, `false`},
 		// strings
-		{"string", `"string"`},
-		{'r', `"114"`}, // rune is an alias of int32
+		{"string", `string`},
+		{'r', `114`}, // rune is an alias of int32
 		// binary
-		{byte(36), `"36"`}, // byte is an alias for uint8
-		{[]byte("hello"), `"hello"`},
+		{byte(36), `36`}, // byte is an alias for uint8
+		{[]byte("hello"), `hello`},
 		// composite => NOT SUPPORTED YET
-		// {stringArray, `"[a b]"`},
+		// {stringArray, `[a b]`},
 		// {struct {
 		// 	a string
 		// 	b string
-		// }{"a", "b"}, `"{a b}"`},
+		// }{"a", "b"}, `{a b}`},
 		// references (slices, maps) => NOT SUPPORTED YET
-		// {[]string{"a", "b"}, `"[a b]"`},
-		// {map[string]string{"k1": "a", "k2": "b"}, `"map[k1:a k2:b]"`},
+		// {[]string{"a", "b"}, `[a b]`},
+		// {map[string]string{"k1": "a", "k2": "b"}, `map[k1:a k2:b]`},
 		// interfaces
 	}
```

Comment:
This change duplicate a test in the file (TestValueMarshalString becomes exactly TestValueFormatString).
So instead of updating the test data, you should modify the call to `value.String()` at line 150 by `value.MarshalJSON()` .